tests/components/application-connector/resources/charts/gateway-test/charts/test/templates/applications/credentials/mtls-oauth-nagative-other-ca.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 type: Opaque
 data:
   {{- $files := .Files }}
-  crt: {{ $files.Get "certs/negative/client.crt" | b64enc  }}
-  key: {{ $files.Get "certs/negative/client.key" | b64enc  }}
-  clientId: {{ "someClientID1" | b64enc }}
+  crt: {{ $files.Get "certs/invalid-ca/client.crt" | b64enc  }}
+  key: {{ $files.Get "certs/invalid-ca/client.key" | b64enc  }}
```

Comment:
Why is the `...negative...` changed to the `...invalid-ca...` there?

Reply:
The same CA signs a positive certificate and a negative certificate. For this test, I generated certificates that are signed by other CA.